src/DependencyInjection/Kfz24QueueExtension.php

```diff
@@ -55,15 +55,21 @@ public function load(array $configs, ContainerBuilder $container)
             $queueClientDefinition = new Definition($clientClass);
             $queueClientDefinition
                 ->addTag('kfz24.queue.client')
-                ->addArgument(new Reference($adapterDefinitionName))
-                ->addArgument($client['resource']);
+                ->setArguments(
+                    [
+                        new Reference($adapterDefinitionName),
+                        new Reference('serializer'),
+                        $client['resource']
+                    ]
+                );
 
             if ($clientType === 'sqs') {
                 $queueClientDefinition->addMethodCall('setValidator', [new Reference('kfz24.queue.message_validator')]);
 
                 if ($client['large_payload_client']['enabled']) {
                     $s3DefinitionName = sprintf('kfz24.queue.s3client.%s', $name);
-                    $largePayloadMessageExtensionDefinitionName = sprintf('kfz24.queue.large_payload_message_extension.%s', $name);
+                    $largePayloadMessageExtensionDefinitionName = sprintf('kfz24.queue.large_payload_message_extension.%s',
```

Comment:
OPTIONAL: We could unwrap the `sprintf` on different lines to format it better

Reply:
+ 1 for this comment.